.github/workflows/build_and_test_snap.yaml

```diff
@@ -64,9 +64,9 @@ jobs:
 
     - name: Export kubeconfig where strict snap can access it
       run: |
-        mkdir -p ~/.dss
-        cp ~/.kube/config ${{ env.KUBECONFIG }}
+        mkdir -p $(dirname ${{ env.DSS_KUBECONFIG_PATH }})
+        cp ~/.kube/config ${{ env.DSS_KUBECONFIG_PATH }}
```

Comment:
I dont think we should be doing this. We expect users to use the dss cli with command 

```
dss initialize --kubeconfig="$(microk8s config)"
```

so I think we should be testing that in this CI

Reply:
Yeah I see where you're coming from on that.  The downsides I see are:

* testing with `dss initialize` means the tests change the cluster's state.  So running them a second time gives a different answer
* `dss status` is faster (doesn't need to pull mlflow, etc)
* if we put `dss initialize` in the tests, we have to pass the kubeconfig through to the tests too.  So instead of `tox -e dss-cli` people always have to call it with `tox -e dss-cli -- --kubeconfig "$(microk8s config)"`

I'm against the change, but the above points are minor so I'm not hard against it.  If you prefer, we can change the tox env and use `dss initialize` as the test instead of `dss status`

Reply:
ok We do the kubeconfig test in integration tests now so its perfect :)